lesson-7/server/api.js

```diff
@@ -0,0 +1,56 @@
+const MongoClient = require('mongodb').MongoClient;
+const config = require('./config');
+
+function executeQuery(query) {
+    return new Promise((resolve, reject) => {
```

Comment:
No need to create new Promise, we can `return MongoClient.connect ...`

Reply:
done